en/components/grids_templates/row_pinning.md

```diff
@@ -434,13 +398,146 @@ This would allow reordering the rows and moving them between the pinned and unpi
+
+ ```scss
+$primary-color: #292826;
+$secondary-color: #ffcd0f;
+
+$custom-palette: igx-palette(
+  $primary: $primary-color,
+  $secondary: $secondary-color
+);
+```
+
+After a custom palette has been generated, the `igx-color` function can be used to obtain different varieties of the primary and the secondary colors.
+
+
+```scss
+$custom-theme: igx-grid-theme(
+    $pinned-border-width: 5px,
+    $pinned-border-style: double,
+    $pinned-border-color: igx-color($custom-palette, "secondary", 500),
+    $cell-active-border-color: igx-color($custom-palette, "secondary", 500)
+);
+```
+
+The `$custom-theme` contains the same properties as the one in the previous section, but this time the colors are not hardcoded. Instead, the custom `igx-palette` was used and the colors were obtained through its primary and secondary colors, with a given color variant.
+
+#### Defining custom schemas
+
+You can go even further and build flexible structure that has all the benefits of a [**schema**](../themes/schemas.md). The **schema** is the recipe of a theme.
+Extend one of the two predefined schemas, that are provided for every component. In our case, we would use `$_light_grid`.
+
+```scss
+$custom-grid-schema: extend($_light-grid,(
+    pinned-border-width: 5px,
+    pinned-border-style: double,
+    pinned-border-color: igx-color:("secondary", 500),
+    cell-active-border-color: igx-color:("secondary", 500)
+));
+```
+In order for the custom schema to be applied, either `light`, or `dark` globals has to be extended. The whole process is actually supplying a component with a custom schema and adding it to the respective component theme afterwards.
+
+```scss
+$my-custom-schema: extend($light-schema, (
+    igx-grid: $custom-grid-schema
+));
+$custom-theme: igx-grid-theme(
+    $palette: $custom-palette,
+    $schema: $my-custom-schema
+);
+```
+
+#### Applying the custom theme
+The easiest way to apply your theme is with a `sass` `@include` statement in the global styles file:
+```scss
+@include igx-grid($custom-theme);
+```
+
+#### Scoped component theme
+
+In order for the custom theme to affect only specific component, you can move all of the styles you just defined from the global styles file to the custom component's style file (including the import of the `index` file).
+
+This way, due to Angular's [ViewEncapsulation](https://angular.io/api/core/Component#encapsulation), your styles will be applied only to your custom component.
+ >[!NOTE]
+ >If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to penetrate this encapsulation using `::ng-deep` in order to style the grid.
+ >[!NOTE]
+ >Wrap the statement inside of a `:host` selector to prevent your styles from affecting elements *outside of* our component:
+
+```scss
+:host {
+    ::ng-deep {
+        @include igx-grid($custom-theme);
+    }
+}
```

Comment:
This example should use the `igx-css-vars` by default as it's the preferred method for including themes. You should mention the `igx-grid` only in the context of IE11. The scoped theme sample should use the igx-css-vars mixin as well. Please, take a look at the avatar topic for instance.